qrenderdoc/Windows/Dialogs/RemoteManager.cpp

```diff
@@ -340,7 +343,7 @@ void RemoteManager::updateConnectButton()
       {
         ui->connect->setText(tr("Run Server"));
 
-        if(host->RunCommand.isEmpty())
+        if(!host->Hostname.startsWith(lit("adb:")) && host->RunCommand.isEmpty())
```

Comment:
Since we're not going to be using the run command it would also be nice to disable editing on the run command textbox for adb hosts (the same as I do for localhost).